<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>221</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Datos De Simulacion</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>31</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Posiciones iniciales</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Masa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Radio</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocidades iniciales</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPosX">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPosY">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMasa">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRadio">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVelX">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVelY">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonIniciarSimulacion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>86</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>INICIAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDetenerSimulacion">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>540</y>
       <width>86</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>PARAR</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxCantidadCuerpos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>45</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad De Cuerpos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAgregarCuerpo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>490</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar Cuerpo </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>si</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
